packages/7z-nsis.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = '7z-NSIS'
```

Comment:
The tool name is `7z` (the file with the `.exe`). That is the tool I would expect in path and what I use in existent scripts. But this would conflict without our current 7z package. I wouldn't install both version of `7z` at the same time, so I think this ok. @mandiant/vms opinions?